test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -283,7 +283,8 @@ def setUp(self):
 }
 
 jobnum=${PBS_JOBID%%.*}
-cpuset_base=`grep cgroup /proc/mounts | grep cpuset | cut -d' ' -f2`
+cpuset_base=`grep cgroup /proc/mounts | grep cpuset | cut -d' ' -f2 | \
+             tr " " "\n" | sed -n '1p'`
```

Comment:
What is tr accomplishing? You are guaranteed there will be no spaces by the fact you're using space as a delimiter in the cut command. If you just want the first mount point, why not do this...
```
cpuset_base=`grep cgroup /proc/mounts | grep cpuset | cut -d' ' -f2 | head -1`
```

Reply:
I wanted to be consistent with what was previously done in cpuset_mem_script (line 213).  I have now changed both cpuset_base assignments in lines 213 and 286.